controllers/consoleplugin/config/static-frontend-config.yaml

```
@@ -526,6 +526,78 @@ columns:
     default: true
     width: 15
     feature: networkEvents
+  - id: XlatZoneId
+    group: Xlat
+    name: Xlat zone id
+    field: ZoneId
+    filter: xlat_zone_id
+    default: true
+    width: 5
+  - id: XlatIcmpId
+    group: Xlat
+    name: Xlat icmp code
+    field: XlatIcmpId
+    filter: xlat_icmp_id
+    default: false
+    width: 5
+    feature: packetTranslation
+  - id: XlatSrcAddr
+    group: Xlat
+    name: Xlat source IP
+    tooltip: The packet translation source IP address. Can be either in IPv4 or IPv6 format.
+    field: XlatSrcAddr
+    filter: xlat_src_address
+    default: false
+    width: 10
+    feature: packetTranslation
+  - id: XlatSrcPort
+    group: Xlat
+    name: Xlat source port
+    tooltip: The packet translation source port number.
+    field: XlatSrcPort
+    filter: xlat_src_port
+    default: false
+    width: 10
+    feature: packetTranslation
+  - id: XlatSrcK8S_Object
+    group: Xlat
+    name: Xlat Src Kubernetes Object
+    calculated: kubeObject(SrcK8S_Type,SrcK8S_Namespace,SrcK8S_Name,1) or concat(XlatSrcAddr,':',XlatSrcPort)
```

Based on how you configured FLP below, I guess it should be this: 

```suggestion
    calculated: kubeObject(XlatSrcK8S_Type,XlatSrcK8S_Namespace,XlatSrcK8S_Name,1) or concat(XlatSrcAddr,':',XlatSrcPort)
```